src/modules/i18n/locales/de.json

```diff
@@ -1 +1 @@
":"Wildruhezonen","wildruhezonen_service_link_href":"http://wildruhezonen.ch/","wildruhezonen_service_link_label":"wildruhezonen.ch","wms":"WMS","wms_max_size_allowed":"Maximal erlaubte WMS Grösse","wmsimport_tooltip":"Importieren Sie externe WMS Datenquellen","yellow":"Gelb","zoom_in":"Vergrössere Kartenausschnitt","zoom_out":"Verkleinere Kartenausschnitt","edit_glstyle_choose_color":"Farbe auswählen","edit_glstyle_choose_layer":"Ebene auswählen","landuse-residential":"Landuse residential","landcover_grass":"Landcover grass","edit_not_possible":"Leider können Sie diese Ebene nicht bearbeiten. Bitte wählen Sie einen anderen Hintergrund aus","edit_current_layer":"Bearbeitung des Datensatz ","edit_fill_color":"Farbe auswählen","edit_text_color":"Textfarbe auswählen","edit_text_size":"Textgrösse auswählen","edit_text_halo_color":"Farbe Schatten  auswählen","show":"Anzeigen","hide":"Deaktivieren","displayed":"Angezeigt","hidden":"Versteckt","edit_fill_outline_color":"Randfarbe auswählen","edit_line_color":"Linienfarbe auswählen","edit_line_width":"Liniendicke auswählen","light_gray":"hellgrau","light_green":"hellgrün","light_blue":"hellblau","light_yellow":"hellgelb","purple":"lila","edit_or_reset_style":"Diese Ebene verwendet einen Stil von Drittanbietern. Sie können diesen nicht ändern, aber Sie können den Standardstil verwenden, indem Sie auf diese Schaltfläche klicken.","basis":"Basiskarte light","labels":"Labels","roadtraffic":"Strassen","hydrology":"Hydrologie","landuse":"Grünflächen","settlement":"Siedlungsgebiet","territory":"Gelände","woodland":"Wald","edit_visibility":"Darstellung","current_value":"Aktueller Wert","mvt_ie11_alert":"Sie verwenden InternetExplorer ≤ 11:  Bitte nutzen Sie die neueste Version von Microsoft EDGE, Firefox oder Chrome, um optimale Ergebnisse zu erhalten","try_test_viewer":"Probieren Sie test.map.geo.admin.ch aus","obstacle_started_last_4_days":"Aktivierte Hindernisse letzte 4 Tage","obstacle_deleted_last_4_days":"Gelöschte Hindernisse letzte 4 Tage","draw_button_delete_last_point":"Letzten Punkt löschen","elevation_href":"https://www.swisstopo.admin.ch/de/wissen-fakten/geodaesie-vermessung/bezugsrahmen/lokal/ln02.html","copy_cta":"Kopieren","copy_done":"Kopiert!","test_map_give_feedback":"Feedback geben","legacy_drawing_warning":"Es ist nicht möglich, eine von map.geo.admin.ch erstellte Zeichnung auf der Website test.map.geo.admin.ch zu bearbeiten. Wenn Sie eine neue Zeichnung erstellen möchten, müssen Sie entweder die aktuelle Zeichnung entfernen oder die Auswahl aufheben.","draw_file_load_error":"Laden von KML fehlgeschlagen","draw_file_save_error":"Speichern von KML fehlgeschlagen","very_small_size":"Sehr klein","feedback_rating_text":"Teilen Sie uns mit, wie Sie sich mit unserem Kartenviewer fühlen","feedback_rating_title":"Wie würden Sie Ihre Erfahrung bewerten ? (erforderlich)","send_failed":"Senden fehlgeschlagen","feedback_email":"Ihre E-Mail Adresse (optional)","feedback_invalid_email":"ungültige E-Mail"}
```

Comment:
Not related to this PR but for review it would be nice to have these json file formatted.

Reply:
that's a good point, I've created a ticket so that we can tackle that later on
https://jira.swisstopo.ch/browse/BGDIINF_SB-2786